source/code/plugin/filter_cadvisor_health_container.rb

```diff
@@ -5,66 +5,57 @@ module Fluent
     require 'logger'
     require 'json'
     require_relative 'oms_common'
-    require_relative 'HealthMonitorUtils'
-    require_relative 'HealthMonitorState'
     require_relative "ApplicationInsightsUtility"
+    Dir[File.join(__dir__, './health', '*.rb')].each { |file| require file }
```

Comment:
I remember we had problems with this earlier. Are we still planning to keep the same bevaiour?